config/settings.yml

```diff
@@ -52,11 +52,11 @@
     :agent_coordinator:
       :agent_ami_name: CentOS Atomic Host 7 x86_64 HVM EBS 1706_01
       :agent_ami_login_user: centos
-      # TODO: will replace with official docker image name
-      :agent_docker_name: docker.io/hsong/ssa_support
+      :agent_ami_registration_required: false
+      :agent_docker_name: manageiq/amazon-smartstate:latest
       :agent_idle_period: 900
       :agent_label: smartstate
-      :agent_log_level: INFO
+      :log_level: INFO
```

Comment:
Is this the log level of the agent coordinator or the fleecing code running on the remote machine?

Reply:
It's the fleecing code running on the remote agent.